task-sdk/src/airflow/sdk/definitions/baseoperator.py

```diff
@@ -901,9 +901,6 @@ def say_hello_world(**context):
         "executor",
     }
 
-    # Defines if the operator supports lineage without manual definitions
-    supports_lineage: bool = False
```

Comment:
There is single operator (`PapermillOperator`) that implements this. Right time to remove this from it would be when Airflow 2 gets out of provider scope, right?